### PR TITLE
Code Review Feedback

### PR DESCRIPTION
**Pull request by efalsken**

### CODE REVIEW

File: src/DDTech.Localization/Alerts.ru.resx [hunk 1]
```diff
@@ -379,13 +379,13 @@
...
   <data name="rule_MaterialSynchronizationAlert_B_Critical2" xml:space="preserve">
-    <value />
+    <value>QOH (ток на наличии) &amp;gt; = 0 и, если баланс работает &amp;lt; = 0 в любое время в течение периода привести</value>
   </data>
   <data name="rule_MaterialSynchronizationAlert_High1" xml:space="preserve">
     <value>Имеет открытый спроса в течение времени</value>
   </data>
   <data name="rule_MaterialSynchronizationAlert_High2" xml:space="preserve">
-    <value />
+    <value>QOH (ток на наличии) &amp;gt; = 0 и работает баланс &amp;gt; 0 и &amp;lt; = % ToR в любое время в течение периода привести</value>
```
Comment (tstafney) [inline src/DDTech.Localization/Alerts.ru.resx]: This happened last time too... Not sure we wanted these two Russian "Alerts" txts changed? They were rolled back last PR.

Comment (efalsken) [inline src/DDTech.Localization/Alerts.ru.resx]: Then again, changed from the blank non-translation that they were. .... are we sure we don't want them changed?

Comment (tynorton) [inline src/DDTech.Localization/Alerts.ru.resx]: Yes, that last handback removed it.

File: src/DDTech.Localization/Alerts.ru.resx [hunk 2]
```diff
@@ -379,13 +379,13 @@
     <value>Есть открытые предложения и спроса в течение времени</value>
   </data>
   <data name="rule_MaterialSynchronizationAlert_B_Critical2" xml:space="preserve">
-    <value />
+    <value>QOH (ток на наличии) &amp;gt; = 0 и, если баланс работает &amp;lt; = 0 в любое время в течение периода привести</value>
```
Comment (tstafney) [inline src/DDTech.Localization/Alerts.ru.resx]: I'll specifically mark this one too. For posterity.

File: src/DDTech.Localization/Alerts.ru.resx (hunk 1 above)
Comment (tstafney) [inline src/DDTech.Localization/Alerts.ru.resx]: Yeah... Not sure. That's just the way it was. I'll let @norton apply the tie-breaker :wink:

File: src/DDTech.Localization/Alerts.ru.resx (hunk 2 above)
Comment (tynorton) [inline src/DDTech.Localization/Alerts.ru.resx]: All I can tell you in the `ru-RU` handback had those lines removed. I assumed intentionally. I'm sure they'll send updated strings if they want it re-added. :)

**Pull request merged**